linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/MethodAndHostIdentifierTest.scala

```diff
@@ -0,0 +1,16 @@
+package io.buoyant.linkerd.protocol.http
+
+import com.twitter.finagle.Path
+import com.twitter.finagle.util.LoadService
+import io.buoyant.linkerd.IdentifierInitializer
+import org.scalatest.FunSuite
+
+class MethodAndHostIdentifierTest extends FunSuite {
+  test("sanity") {
+    val _ = new MethodAndHostIdentifierConfig().newIdentifier(Path.empty)
+  }
+
+  test("service registration") {
+    assert(LoadService[IdentifierInitializer].exists(_.isInstanceOf[MethodAndHostIdentifierInitializer]))
+  }
```

Comment:
These two tests are also present in linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/MethodAndHostIdentifierConfigTest.scala -- maybe just delete this file?